astropy/visualization/wcsaxes/core.py

```diff
@@ -432,7 +432,7 @@ def draw_wcsaxes(self, renderer):
 
         self.coords.frame.draw(renderer)
 
-    def draw(self, renderer, inframe=False):
```

Comment:
Actually thinking about it more, what about just changing this to ``**kwargs`` and passing it on to the draw call below?

Reply:
Great idea, thanks! Done.